fixtures/workers-with-assets/src/index.ts

```diff
@@ -0,0 +1,18 @@
+export interface Env {
+	// Example binding to a Service. Learn more at https://developers.cloudflare.com/workers/runtime-apis/service-bindings/
+	ASSETS: Fetcher;
+}
+
+export default {
```

Comment:
I'm a a little confused by this fixture setup. I'll come back to it, or ping you async with questions

Reply:
nvm my brain was just tired yesterday. Keep calm and carry on and ignore me :)